remote/github/github.go

```diff
@@ -249,6 +256,21 @@ func (g *Github) Status(u *model.User, r *model.Repo, b *model.Build, link strin
 	return err
 }
 
+func deploymentStatus(client *github.Client, r *model.Repo, b *model.Build, link string) error {
+	// the deployment ID is only available in the the link to the build as the last element in the URL
+	parts := strings.Split(b.Link,"/")
+	id, _ := strconv.Atoi(parts[len(parts)-1])
+	status := getStatus(b.Status)
```

Comment:
Reading through the GH API pages, the workflow is what matters.

1. Create GH "Deployment"... Which triggers...
2. A deployment build... Which...
3. Update GH with the status.

The GH deployment has an id thats independent of Drone. When you submit back a status, you need to have this match the deployment.

In the case of a manual trigger, there won't be a GH deployment ID, thus this will fail.

In general I'm guessing this will fail for that reason.